wireless_modules_py/wind_module/config.py

```diff
@@ -0,0 +1,6 @@
+MQTT_BROKER = "localhost"
```

Comment:
Does this need to be turned into the `config.example.py` format that then gets copied to a gitignored `config.py` to make it a bit harder to accidently commit passwords? This is more of a question for the wind sensor project, but putting it here while I remember.